src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,55 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs based on the following:
+  * http://stackoverflow.com/questions/2315912/scala-best-way-to-parse-command-line-parameters-cli
+  * @constructor Create a new command reader
+  */
+object CommandReader {
+  val usage =
+    """
+       Usage: [spark arguments] sparkRocks-assembly-1.0.jar -inputFile [inputFile] -numberSeedJoints [Int] [output opts]
```

Comment:
You'll probably also want to change the syntax of this usage string. Usually if something is in brackets it's optional. Check out some of the man pages for examples. Perhaps angle brackets would be better, so something like `-inputFile <inputFile>`